src/main/java/gregtech/api/recipes/machines/RecipeMapGroupOutput.java

```diff
@@ -40,6 +40,11 @@ public Builder createUITemplate(DoubleSupplier progressSupplier, IItemHandlerMod
         return builder;
     }
 
+    @Override
+    public Builder createUITemplate(DoubleSupplier progressSupplier, IItemHandlerModifiable importItems, IItemHandlerModifiable exportItems, FluidTankList importFluids, FluidTankList exportFluids, int yOffset) {
+        return createUITemplate(progressSupplier, importItems, exportItems, importFluids, exportFluids);
+    }
+
```

Comment:
Isn't this backwards? Shouldn't the non offset one redirect to the offset method with an offset of zero?